pkg/quota_management/account.go

```diff
@@ -1,20 +1,69 @@
 package quota_management
```

Comment:
Are `pkg/quota_management/account.go` and `pkg/quota_management/organisation.go` exclusive to quota management list?
If so I think we should rename the package to be more specific to quota management list or move the contents to a more specific to quota management list package(s) as in that case it gives the idea that is where elements to perform quota management, independently of the implementation, are located.

Reply:
I had exactly the same thought. I already discussed this with Manyanda and we agreed to change that package name.
I will do it in another PR.